scripts/prepare_sector_network.py

```diff
@@ -520,7 +520,8 @@ def add_carrier_buses(n, carrier, nodes=None):
 
     unit = "MWh_LHV" if carrier == "gas" else "MWh_th"
     # preliminary value for non-gas carriers to avoid zeros
-    capital_cost = costs.at["gas storage", "fixed"] if carrier == "gas" else 0.02
```

Comment:
Not sure I understand the connection to the discount rate here, but 14 €/MWh/a is much too high for liquid fuels.